Source/WebCore/page/MemoryRelease.cpp

```diff
@@ -160,6 +160,11 @@ void releaseMemory(Critical critical, Synchronous synchronous, MaintainBackForwa
 {
     TraceScope scope(MemoryPressureHandlerStart, MemoryPressureHandlerEnd, static_cast<uint64_t>(critical), static_cast<uint64_t>(synchronous));
 
+#if PLATFORM(IOS_FAMILY)
```

Comment:
Shouldn't this go in platformReleaseMemory() in the Cocoa file since this is IOS only?

Reply:
Possibly yes. I was trying to put the code in the same file as the other call to `garbageCollectNowIfNotDoneRecently`, which is also scoped to only occurring on IOS_FAMILY.